### PR TITLE
Symantec Management Center - add support for LOCAL_CATEGORY_DB content type

### PR DESCRIPTION
**Pull request by Itay4**
## Status
- [x] Ready

## Related Issues
https://github.com/demisto/etc/issues/18631

## Description
 - Added support for content type `LOCAL_CATEGORY_DB` for the add and delete content commands
 - Added update content command to update content comment/description and enabled flag

## Does it break backward compatibility?
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by Itay4** [comment]
@kirbles19 please review yml and release notes

**Comment by kirbles19** [comment]
@Itay4 Done, no change.

**Comment by liorblob** [comment]
@Itay4 no issue linked? maybe a better description of the use case?

**Comment by Itay4** [comment]
> @Itay4 no issue linked? maybe a better description of the use case?

updated the main comment

File: Packs/SymantecManagementCenter/Integrations/SymantecManagementCenter/SymantecManagementCenter.py [hunk 1]
```diff
@@ -776,8 +776,13 @@ def add_policy_content_command():
     elif content_type == CATEGORY_LIST_TYPE:
         add_policy_content_request(uuid, content_type, change_description, schema_version,
                                    categories=categories)
+    elif content_type == LOCAL_CATEGORY_DB_TYPE:
+        add_policy_content_request(uuid, content_type, change_description, schema_version,
+                                   urls=urls, categories=categories, ips=ips, description=description)
 
-    return_outputs('Successfully added content to the policy', {}, {})
+    output = demisto.args()
```
Comment (Itay4) [inline Packs/SymantecManagementCenter/Integrations/SymantecManagementCenter/SymantecManagementCenter.py:783]: add comment

File: Packs/SymantecManagementCenter/Integrations/SymantecManagementCenter/SymantecManagementCenter.py [hunk 2]
```diff
@@ -907,6 +993,131 @@ def delete_policy_content_request(uuid, content_type, change_description, schema
...
+    if schema_version:
+        body['schemaVersion'] = schema_version
+
+    content = get_policy_content_request(uuid)
+
+    if not content or 'content' not in content:
+        return_error('Could not update policy content - failed retrieving the current content')
+
+    found_object_to_update = False
+
```
Comment (Itay4) [inline Packs/SymantecManagementCenter/Integrations/SymantecManagementCenter/SymantecManagementCenter.py:1077]: change the condition to check if the type is local cat db

File: Packs/SymantecManagementCenter/Integrations/SymantecManagementCenter/SymantecManagementCenter.py [hunk 3]
```diff
@@ -809,7 +814,35 @@ def add_policy_content_request(uuid, content_type, change_description, schema_ve
...
+            for entity in content_entities:
+                entries.append({
+                    'type': 'url',
+                    'url': entity,
+                    'comment': description
+                })
+            found_existing_category = False
+            for index, existing_category in enumerate(content['content']['categories']):
+                if existing_category.get('name') == category:
+                    content['content']['categories'][index]['entries'].extend(entries)
```
Comment (Itay4) [inline Packs/SymantecManagementCenter/Integrations/SymantecManagementCenter/SymantecManagementCenter.py:836]:
```suggestion
                    existing_category['entries'].extend(entries)
```

File: Packs/SymantecManagementCenter/Integrations/SymantecManagementCenter/SymantecManagementCenter.py [hunk 4]
```diff
@@ -856,14 +889,26 @@ def delete_policy_content_command():
...
     elif content_type == URL_LIST_TYPE:
         delete_policy_content_request(uuid, content_type, change_description, schema_version, urls=urls)
     elif content_type == CATEGORY_LIST_TYPE:
         delete_policy_content_request(uuid, content_type, change_description, schema_version, categories=categories)
-
-    return_outputs('Successfully deleted content from the policy', {}, {})
+    elif content_type == LOCAL_CATEGORY_DB_TYPE:
+        delete_policy_content_request(uuid, content_type, change_description, schema_version, ips=ips,
+                                      urls=urls, categories=categories)
+    elif content_type == LOCAL_CATEGORY_DB_TYPE:
```
Comment (Itay4) [inline Packs/SymantecManagementCenter/Integrations/SymantecManagementCenter/SymantecManagementCenter.py:903]: remove

File: Packs/SymantecManagementCenter/Integrations/SymantecManagementCenter/SymantecManagementCenter.py [hunk 5]
```diff
@@ -893,7 +938,48 @@ def delete_policy_content_request(uuid, content_type, change_description, schema
...
+                    if entries_to_keep:
+                        categories_to_keep.append({
+                            'name': category.get('name'),
+                            'type': 'inline',
+                            'entries': entries_to_keep
+                        })
+                    else:
+                        content_deleted.append({
+                            'CategoryName': category.get('name'),
+                            'ObjectDeleted': 'The category was deleted.  Category cannot be empty.'
```
Comment (Itay4) [inline Packs/SymantecManagementCenter/Integrations/SymantecManagementCenter/SymantecManagementCenter.py:975]:
```suggestion
                            'ObjectDeleted': 'The category was deleted. Category cannot be empty.'
```

File: Packs/SymantecManagementCenter/Integrations/SymantecManagementCenter/SymantecManagementCenter.py [hunk 6]
```diff
@@ -907,6 +993,131 @@ def delete_policy_content_request(uuid, content_type, change_description, schema
...
+            uuid, content_type, change_description, schema_version, urls=urls,
+            content_description=content_description, content_enabled=content_enabled
+        )
+    elif content_type == LOCAL_CATEGORY_DB_TYPE:
+        update_policy_content_request(
+            uuid, content_type, change_description, schema_version, ips=ips, urls=urls,
+            categories=categories, content_description=content_description, content_enabled=content_enabled
+        )
+
+    output = demisto.args()
```
Comment (Itay4) [inline Packs/SymantecManagementCenter/Integrations/SymantecManagementCenter/SymantecManagementCenter.py:1041]: add comment

File: Packs/SymantecManagementCenter/Integrations/SymantecManagementCenter/SymantecManagementCenter.py [hunk 7]
```diff
@@ -907,6 +993,131 @@ def delete_policy_content_request(uuid, content_type, change_description, schema
...
+
+    if ips:
+        if categories:
+            if 'categories' in content['content']:
+                for category_index, category in enumerate(content['content']['categories']):
+                    if category.get('name') in categories:
+                        for entry_index, entry in enumerate(category.get('entries')):
+                            if entry.get('url') in ips:
+                                found_object_to_update = True
+                                content['content']['categories'][category_index]['entries'][entry_index][
```
Comment (Itay4) [inline Packs/SymantecManagementCenter/Integrations/SymantecManagementCenter/SymantecManagementCenter.py:1086]: check if enumerate is needed

File: Packs/SymantecManagementCenter/Integrations/SymantecManagementCenter/SymantecManagementCenter_test.py [hunk 8]
```diff
@@ -0,0 +1,297 @@
+from SymantecManagementCenter import main, LOCAL_CATEGORY_DB_TYPE
```
Comment (Itay4) [inline Packs/SymantecManagementCenter/Integrations/SymantecManagementCenter/SymantecManagementCenter_test.py:1]: add test for content not found

**Pull request merged**